src/inference/src/dev/threading/cpu_streams_executor.cpp

```diff
@@ -504,7 +504,12 @@ void CPUStreamsExecutor::cpu_reset() {
 CPUStreamsExecutor::CPUStreamsExecutor(const IStreamsExecutor::Config& config) : _impl{new Impl{config}} {}
 
 CPUStreamsExecutor::~CPUStreamsExecutor() {
-    cpu_reset();
+    try {
+        cpu_reset();
+    } catch (const ov::Exception&) {
+        // Destructor should not throw - catch needed for static analysis.
+        // CPU::CPU() won't throw here as cpu_info() is called from Stream constructor.
```

Comment:
I bet there might be different reason why cpu_reset() might throw an exception? Whatever the reason is shouldn't we abort the execution instead of hiding the error? From my understanding `_cpu_mapping_table` might be in invalid state in case `cpu_reset` throw somewhere in the middle of execution.

Reply:
Yes, you are right, I added an exception thrown.